common/supplemental/windowsZones.xml

```diff
@@ -537,7 +537,8 @@ For terms of use, see http://www.unicode.org/copyright.html
 			<!-- (UTC+05:00) Ashgabat, Tashkent -->
 			<mapZone other="West Asia Standard Time" territory="001" type="Asia/Tashkent"/>
 			<mapZone other="West Asia Standard Time" territory="AQ" type="Antarctica/Mawson"/>
-			<mapZone other="West Asia Standard Time" territory="KZ" type="Asia/Oral Asia/Aqtau Asia/Aqtobe Asia/Atyrau"/>
+			<!-- Microsoft may create a new zone dedicated for Almaty and Qostanay. -->
+			<mapZone other="West Asia Standard Time" territory="KZ" type="Asia/Oral Asia/Almaty Asia/Aqtau Asia/Aqtobe Asia/Atyrau Asia/Qostanay"/>
```

Comment:
Almaty and Qostanay will use UTC+5 with no DST, same as other zones in Kazakhstan. Almaty is the most populous city in Kazakhstan, so it can be the primary zone for the region. However, I think Microsoft eventually need to create a new zone for Almaty/Qostanay to preserve historic UTC offset changes separate from this. So this is a temporary solution. This is the reason why I did not change the primary zone (Asia/Oral) in this PR. When Microsoft decide to add a new Windows zone for Almaty/Qostanay, then we will move them to the new zone.